text.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo">Jo</div>
+            <div></div>
+            <div class="menu">
+                <a href="#" class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </a>
+            </div>
+        </header>
+        <section class="headline">
+            <div></div>
+            <div class="greet">I'm Jonathan Glad to see you!</div> 
+            <div class="presentation">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</div>
```

Comment:
- [ ] I think it will be better if you wrap your headline text description in a `<p>` tag which will make your website semantic which will in turn increase its accessibility 👍🏽